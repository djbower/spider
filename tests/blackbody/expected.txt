description: dS/dr (basic nodes)
scaling: 0.0004697889028520918
values:
val:                0
val: -0.03152234340836896
val: -0.01258130280968732
val: -0.007344653219029169
val: -0.005006853347381194
val: -0.0037152109822216
val: -0.002908205337140632
val: -0.002362081058952947
val: -0.001971272206665377
val: -0.001679579243496172
val: -0.001454653704154891
val: -0.001276625574666812
val: -0.001132653842105872
val: -0.00101422241568007
val: -0.0009153350242376478
val: -0.0008317042338516362
val: -0.000760181492069285
val: -0.0006984042517881379
val: -0.0006446093046581772
val: -0.0005974075289111468
val: -0.0005557083339094614
val: -0.0005186374858960435
val: -0.0004854983673706178
val: -0.0004557304707827112
val: -0.0004288673609695499
val: -0.0004045239360820843
val: -0.000382370743829078
val: -0.0003621441270236387
val: -0.0003436155365412714
val: -0.0003265899052166214
val: -0.0003108979626742597
val: -0.0002963965503898916
val: -0.0002829633831853935
val: -0.0002704906540196371
val: -0.0002588827448580301
val: -0.0002480563205539502
val: -0.0002379403375123719
val: -0.0002284708063623567
val: -0.000219590157919111
val: -0.0002112468788225633
val: -0.0002033972717112019
val: -0.0001960010797374073
val: -0.0001890216467395951
val: -0.0001824258477567176
val: -0.0001761853344804925
val: -0.0001702736369445247
val: -0.0001646661671801674
val: -0.0001593412677539593
val: -0.0001542795711873434
val: -0.0001494628866275061
val: -0.0001448743900111791
val: -0.0001404991123554281
val: -0.0001363235598783981
val: -0.0001323350521231495
val: -0.0001285212473949891
val: -0.0001248720865354878
val: -0.0001213776181693843
val: -0.0001180283910634393
val: -0.0001148160313037299
val: -0.0001117328626000817
val: -0.0001087716215440148
val: -0.0001059251880818313
val: -0.0001031876550719934
val: -0.0001005531875048279
val: -9.801605484779484e-05
val: -9.557136098213525e-05
val: -9.32144706419084e-05
val: -9.094084654003159e-05
val: -8.874617121490042e-05
val: -8.662679097454603e-05
val: -8.457904987393902e-05
val: -8.259927696349443e-05
val: -8.068449600552027e-05
val: -7.883170495261618e-05
val: -7.703781690707041e-05
val: -7.53003506671852e-05
val: -7.361678986748662e-05
val: -7.198461244181569e-05
val: -7.040166409405406e-05
val: -6.886585095307412e-05
val: -6.73750440033922e-05
val: -6.592743629828443e-05
val: -6.452123863139225e-05
val: -6.315463467813051e-05
val: -6.182612112533975e-05
val: -6.053413936661023e-05
val: -5.927715422501342e-05
val: -5.805388158421591e-05
val: -5.686295286456255e-05
val: -5.570309314900587e-05
val: -5.457318361908565e-05
val: -5.347208904906353e-05
val: -5.239864573579017e-05
val: -5.135188249355319e-05
val: -5.033078025855829e-05
val: -4.933443882348181e-05
val: -4.836198666136318e-05
val: -4.741247946005965e-05
val: -4.64851622198953e-05
val: -4.557925422329401e-05
val: -4.529693459710531e-05
val: -4.56266531030747e-05
val: -4.596178319431178e-05
val: -4.630248298724159e-05
val: -4.664890638287977e-05
val: -4.70011463905499e-05
val: -4.73593678931057e-05
val: -4.772370953924264e-05
val: -4.809432486602218e-05
val: -4.847139883232621e-05
val: -4.885504033499193e-05
val: -4.924545484420143e-05
val: -4.964280640480932e-05
val: -5.004727330166525e-05
val: -5.045906415954667e-05
val: -5.08783159436789e-05
val: -5.130528405307597e-05
val: -5.174015380049659e-05
val: -5.218313991047925e-05
val: -5.263445674309979e-05
val: -5.309433518563314e-05
val: -5.356304337240783e-05
val: -5.404080130952324e-05
val: -5.452790050384874e-05
val: -5.502458511721556e-05
val: -5.553110011559297e-05
val: -5.604776721096606e-05
val: -5.657487621095827e-05
val: -5.711276660221406e-05
val: -5.766174697565286e-05
val: -5.822213866213279e-05
val: -5.879429501022836e-05
val: -5.937862933983171e-05
val: -5.997555639590809e-05
val: -6.058556041853228e-05
val: -6.120924019916957e-05
val: -6.184733240813683e-05
val: -6.250103581687159e-05
val: -6.317219875192387e-05
val: -6.386409898556651e-05
val: -6.458266656356503e-05
val: -6.533910583656308e-05
val: -6.615466849790124e-05
val: -6.707023985514629e-05
val: -6.816530583949687e-05
val: -6.959370509697718e-05
val: -7.165473702327905e-05
val: -7.493560092842345e-05
val: -8.06247398816194e-05
val: -9.115000977444885e-05
val: -0.0001116188244343627
val: -0.0001527346816615559
val: -0.0002362736106720061
val: -0.0004029894975582781
val: -0.000727115934082164
val: -0.001342982712938425
val: -0.002496672652830748
val: -0.004617201985457709
val: -0.008417569221512572
val: -0.01495690585230369
val: -0.02555519250398465
val: -0.04120371745061577
val: -0.06164249445777788
val: -0.08483834082457599
val: -0.1079176312132618
val: -0.1284934402464451
val: -0.1452988807295271
val: -0.158345456092135
val: -0.1682003031575721
val: -0.1755862076256544
val: -0.1810137533216848
val: -0.1849602136247625
val: -0.1876972774324292
val: -0.1897583326513344
val: -0.1912215738866067
val: -0.1922438240458026
val: -0.1928512152524151
val: -0.1932571171633663
val: -0.19354027434443
val: -0.1936923915028186
val: -0.1937412527806894
val: -0.1935924993699063
val: -0.1935251416039972
val: -0.1934184400761425
val: -0.1932826283197648
val: -0.1930793993370645
val: -0.1928188212890889
val: -0.1926350948044548
val: -0.1924426199522742
val: -0.1922437723736321
val: -0.191976800456337
val: -0.1917073814070939
val: -0.191500362283034
val: -0.1912926934275923
val: -0.1910860319647008
val: -0.1908226616051944
val: -0.1905731977890617
val: -0.1904273887141317
val: -0.1905713028297765
val:                0
description: S at surface
scaling: 0.0004697889028520918
values:
val: 0.8350790064172988
description: Magma Ocean CO2 content
scaling: 0.0004697889028520918
values:
val:                0
description: Magma Ocean H2O content
scaling: 0.0004697889028520918
values:
val:                0
