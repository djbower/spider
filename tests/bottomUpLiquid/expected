%Vec Object:dSdr_b_aug 1 MPI processes
%  type: seq
dSdr_b_aug = [
0.0000000000000000e+00
0.0000000000000000e+00
9.4403973130231089e-01
0.0000000000000000e+00
-2.7949624821443159e-02
-1.0857688213145778e-02
-6.1981965449853897e-03
-4.2197475309493342e-03
-3.1786438730773896e-03
-2.5409508220561173e-03
-2.1094639026961221e-03
-1.7696288632680576e-03
-1.5177372694360344e-03
-1.3250949877526570e-03
-1.1669685374091799e-03
-1.0387796625681452e-03
-9.3378775922972894e-04
-8.4518605931656408e-04
-7.6939927444561547e-04
-7.0477619452396806e-04
-6.4892556776025012e-04
-5.9962380348549274e-04
-5.5644975444492026e-04
-5.1838922564969637e-04
-4.8417558474252333e-04
-4.5359743623665224e-04
-4.2617221076243886e-04
-4.0129063681616404e-04
-3.7866875907248155e-04
-3.5811269648282958e-04
-3.3930981013360488e-04
-3.2199925396342007e-04
-3.0610748513017097e-04
-2.9146681513602605e-04
-2.7785894400244633e-04
-2.6526278519076554e-04
-2.5357607152292819e-04
-2.4265069851689582e-04
-2.3245958385281404e-04
-2.2294334304186455e-04
-2.1401088929914598e-04
-2.0562254005495372e-04
-1.9774718128134878e-04
-1.9032729492503354e-04
-1.8332105754298648e-04
-1.7671279826637399e-04
-1.7046541060087379e-04
-1.6453935457048520e-04
-1.5892751291396776e-04
-1.5360549166350658e-04
-1.4853776736921539e-04
-1.4372193118939195e-04
-1.3914071027484057e-04
-1.3476592828846591e-04
-1.3059481933153582e-04
-1.2661510538560980e-04
-1.2280652376859270e-04
-1.1916399646973761e-04
-1.1567938256065061e-04
-1.1233783897157914e-04
-1.0913336948605498e-04
-1.0606050893383865e-04
-1.0310818754326717e-04
-1.0027026773515643e-04
-9.7543029164947622e-05
-9.4918071490167114e-05
-9.2389528778203921e-05
-8.9954827191550987e-05
-8.7607482754399705e-05
-8.5342105030212186e-05
-8.3156877240324722e-05
-8.1046705755228777e-05
-7.9006768190913179e-05
-7.7035744439531338e-05
-7.5129561972172863e-05
-7.3283988060262741e-05
-7.1498026927927321e-05
-6.9768364900358939e-05
-6.8091340931669420e-05
-6.6466169201719080e-05
-6.4890106646873973e-05
-6.3360025214215877e-05
-6.1875273340816493e-05
-6.0433542730111628e-05
-5.9032192002551499e-05
-5.7670659734103063e-05
-5.6346990500386815e-05
-5.5059030790628222e-05
-5.3806428338263214e-05
-5.2587935239382584e-05
-5.1403033885137029e-05
-5.0254874093214994e-05
-4.9151818005154177e-05
-4.8118978154152994e-05
-4.7226856437974065e-05
-4.6660085024257299e-05
-4.6909073026526484e-05
-4.9283863560643474e-05
-5.7399876917188560e-05
-8.1786551450092722e-05
-1.5147290160853443e-04
-3.3666124823763361e-04
-8.0028075108787495e-04
-1.9141411248624540e-03
-4.5126620388814898e-03
-1.0330694372777324e-02
-2.2390141674229327e-02
-4.4443885678575046e-02
-7.7402928041089392e-02
-1.1586172863720971e-01
-1.5160185014276012e-01
-1.7911488426037350e-01
-1.9828818365636833e-01
-2.1094573817731177e-01
-2.1861391258485241e-01
-2.2338427924932852e-01
-2.2575002814913017e-01
-2.2679681838922244e-01
-2.2689131104278643e-01
-2.2621693935522977e-01
-2.2535472511234175e-01
-2.2389351619301454e-01
-2.2254795623410292e-01
-2.2089209689547185e-01
-2.1915148267244161e-01
-2.1753948944595361e-01
-2.1553332836725911e-01
-2.1392776264612598e-01
-2.1193258612345370e-01
-2.1020752453692515e-01
-2.0832020385093467e-01
-2.0646336830332157e-01
-2.0465959018789950e-01
-2.0274098259404091e-01
-2.0103337588908857e-01
-1.9900630288407054e-01
-1.9737134496593089e-01
-1.9535091566920429e-01
-1.9364648235537771e-01
-1.9174843820416310e-01
-1.8994747596287961e-01
-1.8813800758886545e-01
-1.8630175029903087e-01
-1.8453038417419049e-01
-1.8266243484513661e-01
-1.8092762795770748e-01
-1.7900604947560461e-01
-1.7735115312532174e-01
-1.7537521144585300e-01
-1.7375869490704787e-01
-1.7178880377501030e-01
-1.7017625164999386e-01
-1.6819539751250573e-01
-1.6659024725609034e-01
-1.6459080935381756e-01
-1.6302502918819403e-01
-1.6100668423585821e-01
-1.5944056297732692e-01
-1.5745599059180454e-01
-1.5586269225125454e-01
-1.5389678275992055e-01
-1.5227793573517723e-01
-1.5032112091553276e-01
-1.4870694317114458e-01
-1.4675762623798139e-01
-1.4511390435145546e-01
-1.4322210693495782e-01
-1.4151065573911162e-01
-1.3968820934421139e-01
-1.3791104255348041e-01
-1.3612177242167237e-01
-1.3430910234850554e-01
-1.3255711266904063e-01
-1.3069097572833199e-01
-1.2901074434965082e-01
-1.2704877737388515e-01
-1.2541691978300359e-01
-1.2345291109814815e-01
-1.2176769136362620e-01
-1.1988611814587120e-01
-1.1808560525870948e-01
-1.1631265190855831e-01
-1.1439403404118634e-01
-1.1271478175342427e-01
-1.1070311257345415e-01
-1.0900214998416011e-01
-1.0708772505905334e-01
-1.0527297779840675e-01
-1.0345788606836225e-01
-1.0152554621664008e-01
-9.9799929561104436e-02
-9.7761881519334140e-02
-9.6001104915451288e-02
-9.4087132969989259e-02
-9.2186202356518518e-02
-9.0396723590681488e-02
-8.8341445107641511e-02
-8.6552138970752812e-02
-8.4633462487019584e-02
-8.2716748526558387e-02
-8.1022692380213382e-02
-7.9585927533185261e-02
0.0000000000000000e+00
];
