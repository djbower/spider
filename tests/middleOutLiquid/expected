%Vec Object:dSdr_b_aug 1 MPI processes
%  type: seq
dSdr_b_aug = [
0.0000000000000000e+00
0.0000000000000000e+00
1.0064887684086159e+00
0.0000000000000000e+00
-3.2746643704680181e-02
-1.2360776957545123e-02
-6.8601836862870044e-03
-4.6123766651840151e-03
-3.4650585225931355e-03
-2.7615012546119044e-03
-2.2871061883154289e-03
-1.9215365665619314e-03
-1.6508102647814625e-03
-1.4440286891675683e-03
-1.2734876411175026e-03
-1.1351451394700687e-03
-1.0218770094187341e-03
-9.2624605031800175e-04
-8.4439171522727606e-04
-7.7460568572227306e-04
-7.1429932650468077e-04
-6.6104851925211257e-04
-6.1442037350493365e-04
-5.7332021790553942e-04
-5.3637350646574709e-04
-5.0335432821734991e-04
-4.7374268763103532e-04
-4.4688023864289568e-04
-4.2245871263124933e-04
-4.0026931205958165e-04
-3.7997487639633294e-04
-3.6129278038739922e-04
-3.4414344415159423e-04
-3.2834616824212301e-04
-3.1366497515582117e-04
-3.0007667152226106e-04
-2.8747111604619959e-04
-2.7568831827119234e-04
-2.6469870013880805e-04
-2.5443835937243676e-04
-2.4480890507003715e-04
-2.3576722636134114e-04
-2.2727994138468135e-04
-2.1928493494156447e-04
-2.1173676558713321e-04
-2.0461876815185335e-04
-1.9789090099754243e-04
-1.9151012711970302e-04
-1.8546907146810759e-04
-1.7974146703016842e-04
-1.7428851564580352e-04
-1.6910799009179622e-04
-1.6418133111575328e-04
-1.5947766750249348e-04
-1.5499434869881625e-04
-1.5071822661191707e-04
-1.4662706707234288e-04
-1.4271558741803437e-04
-1.3897515658333278e-04
-1.3538947097954375e-04
-1.3195214038385801e-04
-1.2865746446766467e-04
-1.2549327915053633e-04
-1.2245294716281495e-04
-1.1953268414043664e-04
-1.1672325969900465e-04
-1.1401825939529376e-04
-1.1141515807724494e-04
-1.0890683316923352e-04
-1.0648733557721902e-04
-1.0415496888353471e-04
-1.0190414123804764e-04
-9.9729484417357335e-05
-9.7629843019676090e-05
-9.5600749214072637e-05
-9.3637452742047179e-05
-9.1739147454998336e-05
-8.9902199534412153e-05
-8.8122469326549637e-05
-8.6399372205656118e-05
-8.4729900740122146e-05
-8.3110497955699158e-05
-8.1540724777708480e-05
-8.0018070693414601e-05
-7.8539568337417386e-05
-7.7104981926668948e-05
-7.5712415089222607e-05
-7.4359886300454819e-05
-7.3048240761574695e-05
-7.1777664976789617e-05
-7.0549826214277527e-05
-6.9371607637330911e-05
-6.8252393377862521e-05
-6.7209036703204142e-05
-6.6273046963898571e-05
-6.5484888368733717e-05
-6.4912929446502230e-05
-6.4659304173552223e-05
-6.4836615490980584e-05
-6.5662347196986435e-05
-6.7370264417573937e-05
-7.0193776041300445e-05
-7.4727612735455396e-05
-8.1995547827550064e-05
-9.2525935537582813e-05
-1.0638662991792912e-04
-1.2325462691105792e-04
-1.4449530157710345e-04
-1.6949671321190497e-04
-1.9717095605166629e-04
-2.2944746110065543e-04
-2.6171991228888012e-04
-2.9588768984484032e-04
-3.2939007715291382e-04
-3.5664328891659575e-04
-3.8399545963484579e-04
-3.9955797313261191e-04
-4.1028784129911426e-04
-4.1124084675138927e-04
-3.9741726483826683e-04
-3.8085525316793902e-04
-3.4407384250118836e-04
-3.0723218327851335e-04
-2.5838195626580538e-04
-2.0474308202856218e-04
-1.5378645195755167e-04
-9.8029769833068365e-05
-6.0202956693065823e-05
-3.1000337865418092e-05
-1.8524787914765622e-05
-1.2029631658753711e-05
-8.9396772853422714e-06
-7.4250516355217666e-06
-6.4884826056285985e-06
-6.2635582507827351e-06
-6.1145642664716784e-06
-6.4956077506446976e-06
-6.9558481018592923e-06
-7.8840624664665403e-06
-9.0745026669728464e-06
-1.0800108027995461e-05
-1.3089085263484240e-05
-1.6114693153181217e-05
-2.0108726344540271e-05
-2.5082357399324330e-05
-3.1350411808853853e-05
-3.8603264695005765e-05
-4.7050665337050020e-05
-5.6041985285237771e-05
-6.5676057536500005e-05
-7.5248377876765260e-05
-8.4877983381583108e-05
-9.4055832506102839e-05
-1.0295257522260460e-04
-1.1129323653716332e-04
-1.1925931000729131e-04
-1.2674669890606996e-04
-1.3391496937680792e-04
-1.4075858957328738e-04
-1.4740591265348496e-04
-1.5389781515986331e-04
-1.6033450837524746e-04
-1.6677480372981611e-04
-1.7329765993606951e-04
-1.7996669527323118e-04
-1.8684804172076306e-04
-1.9400549409573828e-04
-2.0150146630280275e-04
-2.0939993774652554e-04
-2.1776669964051608e-04
-2.2667100045146022e-04
-2.3618769584935801e-04
-2.4639860063674872e-04
-2.5739444141495085e-04
-2.6927781798107567e-04
-2.8216649599054311e-04
-2.9619412488204584e-04
-3.1151964958787666e-04
-3.2832565671548524e-04
-3.4683242836908645e-04
-3.6730064491622425e-04
-3.9004498078225113e-04
-4.1544970394473673e-04
-4.4398447473346979e-04
-4.7623535270642324e-04
-5.1293713299231170e-04
-5.5502487059731276e-04
-6.0370844015531508e-04
-6.6057311567681840e-04
-7.2774088993735866e-04
-8.0810896295656646e-04
-9.0572889139566340e-04
-1.0264390646796098e-03
-1.1789305648023587e-03
-1.3766939428830718e-03
-1.6417495235523188e-03
-2.0124084208475554e-03
-2.5611731448595462e-03
-3.4419124723871781e-03
-5.0411716341884193e-03
-8.6390020162007159e-03
-2.1725326243623621e-02
0.0000000000000000e+00
];
