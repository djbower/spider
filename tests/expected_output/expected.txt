description: dS/dr (basic nodes)
scaling: 0.0004697889028520918
val:                0
val: -0.03113052058185595
val: -0.01242414658254165
val: -0.007252591584163371
val: -0.004943929835876311
val: -0.003668422767023934
val: -0.002871519102353702
val: -0.002332243439794877
val: -0.001946342710018447
val: -0.001658318053499864
val: -0.001436224707744835
val: -0.001260440937853717
val: -0.001118286376002376
val: -0.001001351217691863
val: -0.0009037143253572247
val: -0.0008211420248386517
val: -0.000750525292723288
val: -0.0006895312774265335
val: -0.0006364188457318762
val: -0.0005898163474352154
val: -0.0005486469164338676
val: -0.0005120473437303727
val: -0.0004793297828491501
val: -0.0004499407720795922
val: -0.0004234197606927462
val: -0.0003993865074453314
val: -0.0003775157343032013
val: -0.0003575470988401232
val: -0.0003392549397480525
val: -0.0003224466570610938
val: -0.0003069551264650574
val: -0.0002926389998621647
val: -0.0002793775364914132
val: -0.0002670642953379892
val: -0.0002556048716782053
val: -0.0002449169883592578
val: -0.0002349304995187606
val: -0.0002255822279319921
val: -0.0002168153357770844
val: -0.0002085789650631242
val: -0.0002008299714351777
val: -0.0001935286097323499
val: -0.0001866386901379967
val: -0.000180127509437824
val: -0.00017396707889819
val: -0.000168131265445496
val: -0.0001625957943473037
val: -0.0001573392839843854
val: -0.0001523426131653517
val: -0.0001475878228109539
val: -0.0001430583034594544
val: -0.0001387392767710672
val: -0.0001346174210014863
val: -0.0001306802174674423
val: -0.0001269154818647812
val: -0.00012331328343892
val: -0.0001198637978583014
val: -0.0001165576944840188
val: -0.000113386706352198
val: -0.000110343255313227
val: -0.0001074201707846614
val: -0.000104610423477498
val: -0.0001019081820140117
val: -9.930768601722359e-05
val: -9.680327802473777e-05
val: -9.439012469556865e-05
val: -9.206365019397515e-05
val: -8.981937518958016e-05
val: -8.765303737893951e-05
val: -8.556103010638656e-05
val: -8.353974421734761e-05
val: -8.158555552640553e-05
val: -7.96955262590961e-05
val: -7.786669285970321e-05
val: -7.60960076559545e-05
val: -7.43810216214866e-05
val: -7.271925068233071e-05
val: -7.110820482346578e-05
val: -6.954575768716171e-05
val: -6.802984232104298e-05
val: -6.65583567525662e-05
val: -6.512951752701715e-05
val: -6.374155845697561e-05
val: -6.239268626184499e-05
val: -6.108141727058067e-05
val: -5.98062129033926e-05
val: -5.856555747454511e-05
val: -5.735818370183811e-05
val: -5.61827405401008e-05
val: -5.503796934634238e-05
val: -5.392276596747453e-05
val: -5.283600973513745e-05
val: -5.177655183130462e-05
val: -5.074343379384151e-05
val: -4.973564955621606e-05
val: -4.875231063454083e-05
val: -4.779255687040305e-05
val: -4.685545589705153e-05
val: -4.59402627158041e-05
val: -4.504620669920583e-05
val: -4.476844205117208e-05
val: -4.509560526832026e-05
val: -4.542816623956446e-05
val: -4.576628344735808e-05
val: -4.611011136791844e-05
val: -4.645974416054466e-05
val: -4.681534752310555e-05
val: -4.717706110048546e-05
val: -4.754503946534558e-05
val: -4.791946854065962e-05
val: -4.830045894867504e-05
val: -4.868821730139697e-05
val: -4.908290922311071e-05
val: -4.948471467204033e-05
val: -4.989384395954716e-05
val: -5.031043648614949e-05
val: -5.073474939177034e-05
val: -5.116697049468848e-05
val: -5.160731712890077e-05
val: -5.205600660689682e-05
val: -5.251327275060224e-05
val: -5.297938688155032e-05
val: -5.345457278522305e-05
val: -5.393912568320158e-05
val: -5.443329422180598e-05
val: -5.493732837691893e-05
val: -5.5451554584916e-05
val: -5.597626800124081e-05
val: -5.651181361069597e-05
val: -5.705850566017126e-05
val: -5.761667109296645e-05
val: -5.818666702580915e-05
val: -5.876890697884893e-05
val: -5.936379850394886e-05
val: -5.997180302349127e-05
val: -6.059346472497725e-05
val: -6.122940256005492e-05
val: -6.188056898574864e-05
val: -6.254831996751755e-05
val: -6.323495372687524e-05
val: -6.394446262605331e-05
val: -6.468424468031698e-05
val: -6.546815713058168e-05
val: -6.632270448831155e-05
val: -6.729915626470863e-05
val: -6.849637703976997e-05
val: -7.010569899345171e-05
val: -7.249935996659802e-05
val: -7.642078959644308e-05
val: -8.336094045194074e-05
val: -9.64049557457585e-05
val: -0.0001220332740035771
val: -0.0001738868756846778
val: -0.0002786378183640759
val: -0.0004856665406919634
val: -0.0008841462541286997
val: -0.001638290018420705
val: -0.003040467944588494
val: -0.005596977399318963
val: -0.01012125114967209
val: -0.01778925549832397
val: -0.0298839423130413
val: -0.04710254733095034
val: -0.0686254805906569
val: -0.09207635808893876
val: -0.114561263094377
val: -0.1340192124529895
val: -0.1495661009862957
val: -0.1615525686582848
val: -0.1706307204997419
val: -0.1773546003526655
val: -0.1822763851506324
val: -0.1857414693049699
val: -0.1883410898389891
val: -0.1902025307078412
val: -0.1915189002638039
val: -0.1923439668814809
val: -0.1929107340534271
val: -0.1933128168718715
val: -0.1935528476537
val: -0.1936668072390017
val: -0.1935662442899666
val: -0.1935346246393971
val: -0.193454394482475
val: -0.1933381717609433
val: -0.1931494074901871
val: -0.1928994744488662
val: -0.1927236636530191
val: -0.1925370070632153
val: -0.1923424148306086
val: -0.1920784682706239
val: -0.1918112018685016
val: -0.1916057420638776
val: -0.1913991225185422
val: -0.191193099417119
val: -0.1909299290994721
val: -0.1906802036970591
val: -0.1905331500339106
val: -0.1906697632624196
val:                0
description: S at surface
scaling: 2993.025100070677
val: 0.8357038215572162
description: Magma ocean volatile content
scaling: 100
val:                0
val:                0
description: Magma ocean reaction accumulated amounts
scaling: 1.499085657717868e+21
